frontend-cert/css-projects/business-card/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Digital Business Card</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="business-card">
+        <img src="flower.jpg" alt="Profile Image" class="profile-image">
+        <h1>Your Name</h1>
+        <h2>Software Developer</h2>
+        <h3>@freeCodeCamp</h3>
+        <hr>
+        <p>Email: <a href="mailto:your-email@example.com">your-email@example.com</a></p>
+        <p>Phone: (123) 456-7890</p>
+        <p>Portfolio: <a href="http://example.com">example.com</a></p>
+        <hr>
+        <div class="social-media">
+            <p>Connect with me</p>
```

Comment:
```suggestion
            <h2>Connect with me</h2>
```

This is acting as a heading for the three links below it and thus should be marked up as a heading.